docs/introduction/how-tos/quickstart.md

```diff
@@ -1,6 +1,15 @@
 Quickstart Guide
 ================
 
+- [1. Compatibility](#compatibility)
+- [2. Local Install](#local-install)
+
+Compatibility 
+----------------
+
+Local install is not feasible for MacBooks with M1/M2 Apple Sillicone chips. 
```

Comment:
I prefer to skip this part. Currently we depend on a storm docker image that is not built for these processors.

Reply:
Got it!